pkg/driver/controller.go

```diff
@@ -569,6 +559,8 @@ func (cs *controller) GetCapacity(
 		}
 	}
 
+	// round off the available capacity to nearest Mi
+	availableCapacity = (availableCapacity / Mi) * Mi
```

Comment:
we should ceil to the nearest Mi.

Reply:
nope, this is available capacity eventually to be used by kube-scheduler. Making it ceil will give false indication about min capacity available on node.

Reply:
hmmm, let us put a comment about why we are doing a floor here.

Reply:
Sure, done!